whois-commons/src/main/java/net/ripe/db/whois/common/clientauthcertificates/ClientAuthCertificate.java

```diff
@@ -0,0 +1,62 @@
+package net.ripe.db.whois.common.clientauthcertificates;
+
+import net.ripe.db.whois.common.DateTimeProvider;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.dao.jdbc.JdbcRpslObjectSlaveDao;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslAttribute;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ClientAuthCertificate {
```

Comment:
Calling the class `Certificate` suggests it's a domain class. Can you add a noun to clarify the use of this class e.g. `Validator` ?

Reply:
Validator is a bit weird because we have Validators classes for BusinessRuleValidators
but I will try to find a better name

Reply:
I can not get to a better name